substrate/frame/collective/src/lib.rs

```diff
@@ -759,7 +790,10 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		if approve {
 			if position_yes.is_none() {
-				voting.ayes.push(who.clone());
+				voting
+					.ayes
+					.try_push(who.clone())
+					.expect("Proposal voting ayes can't overflow; qed");
```

Comment:
Why can't they overflow?

Reply:
the bound was previously enforced via code, since there could be at most `members_count` votes, and `members_count` is at most `MaxMembers`